test/onnx/dynamo/test_registry_dispatcher.py

```diff
@@ -191,304 +94,330 @@ def setUp(self):
             self.registry, self.diagnostic_context
         )
 
-        self.op_overload = torch.fx.Node(
+    @parameterized.expand(
+        [
+            (
+                torch.fx.Node(
+                    graph=torch.fx.Graph(),
+                    name="aten::add.Tensor",
+                    op="call_function",
+                    target=torch.ops.aten.add.Tensor,  # type: ignore[attr-defined]
+                    args=(torch.tensor(3), torch.tensor(4)),
+                    kwargs={},
+                ),
+                ("aten", "add", "Tensor"),
+            ),
+            (
+                torch.fx.Node(
+                    graph=torch.fx.Graph(),
+                    name="aten::sym_size",
+                    op="call_function",
+                    target=torch.ops.aten.sym_size,
+                    args=(),
+                    kwargs={},
+                ),
+                ("aten", "sym_size", None),
+            ),
+            (
+                torch.fx.Node(
+                    graph=torch.fx.Graph(),
+                    name="builtin_add",
+                    op="call_function",
+                    target=operator.add,
+                    args=(1, 2),
+                    kwargs={},
+                ),
+                ("aten", "add", None),
+            ),
+        ]
+    )
+    def test_get_aten_name_on_supported_fx_node(self, node, expected_name):
+        self.assertEqual(
+            self.dispatcher.get_aten_name(node, self.diagnostic_context),
+            expected_name,
+        )
+
+    @parameterized.expand(
+        [
+            (
+                torch.fx.Node(
+                    graph=torch.fx.Graph(),
+                    name="aten::add",
+                    op="call_function",
+                    target=torch.ops.aten.add,
+                    args=(),
+                    kwargs={},
+                ),
+            ),
+            (
+                torch.fx.Node(
+                    graph=torch.fx.Graph(),
+                    name="builtin_add",
+                    op="call_function",
+                    target=operator.add,
+                    args=("A", "B"),
+                    kwargs={},
+                ),
+            ),
+            (
+                torch.fx.Node(
+                    graph=torch.fx.Graph(),
+                    name="aten::made_up_node",
+                    op="call_function",
+                    target=lambda: None,
+                    args=(),
+                    kwargs={},
+                ),
+            ),
+        ]
+    )
+    def test_get_aten_name_on_unsupported_fx_node(self, node):
+        with self.assertRaises(RuntimeError):
+            self.dispatcher.get_aten_name(node, self.diagnostic_context)
+
+    def test_get_function_overloads(self):
```

Comment:
Include the expected behavior of the test here in the name as well?

Reply:
DOne